debian/rules

```diff
@@ -3,7 +3,11 @@
 VER := $(shell dpkg-parsechangelog | sed -n 's/^Version: *\([^-]\+\)-.\+/\1/p')
 
 %:
-	dh $@ --with python2 --with sphinxdoc
+	dh $@ --with python2,python3 --with sphinxdoc
+
+override_dh_auto_install:
+	python setup.py install --root debian/python-mote --install-layout=deb
+	python3 setup.py install --root debian/python3-mote --install-layout=deb
```

Comment:
I wonder what is this `python-mote`. Did you copy-paste from https://github.com/pimoroni/mote/blob/master/python/packaging/debian/rules ?
If so, I guess it should be changed to "odemis".
However, I think that for Odemis this is a little special. I'd rather avoid having two packages "python-odemis" and "python3-odemis". I think we should still only have a single "odemis" package. Maybe that one includes both the python 2 and python 3 files (so that's it really easy to switch, by changing just the `PYTHON_INTERPRETER` variable. But all the files should go to the same package, so probably `--root debian/odemis`. 

I think there are other alternatives:
* Always package only for one Python version. I'd be fine with only packaging for Python 2. (but still add the Python 3 dependencies, so that it's easy on a computer to switch to the development version on Python 3).
* Have 3 packages: odemis, which contains the generic files and depends on one of the next packages. python-odemis, which contains the Python files for Python 2, and lastly the python3-odemis, which is equivalent for Python 3.

I think this last alternative is nice, but a lot of work for not much usefulness. So I'd recommend to attempt packaging as you're planning now, and if it doesn't work, just switch to Python 2-only.